src/main/resources/META-INF/plugin.xml

```diff
@@ -8,6 +8,7 @@
     <depends>Git4Idea</depends>
 
     <extensions defaultExtensionNs="com.intellij">
+<!--        todo (#825): drop support for 2021.1 -->
```

Comment:
Pls add a comment next to the shortcut that `control` in `$default` keymap means `cmd` on Mac so that nobody scratches their head in future